tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+import shutil
+
+from pytest import fixture
+
+
+@fixture()
+def test_folder():
+    temp_file = "tests/temp_files"
+    if not os.path.exists(temp_file):
+        os.mkdir(temp_file)
+    yield temp_file
+    shutil.rmtree(temp_file)
```

Comment:
See my comment [here](https://github.com/City-of-Helsinki/django-oikotie/pull/17#discussion_r634062838).